### DISCUSSION TITLE
How to change in ButtonStyle

### DISCUSSION THREAD

**Discussion by NafissaBHJ** [bdlukaa/fluent_ui#601] [Q&A]
In the DropDownButton I want to change the background color to transparent, but I can't change the buttonStyle as the ButtonStyle<state> doesn't change, or am I missing on something?
thank you

**Comment by bdlukaa** [comment]
You can wrap your `DropDownButton` with a `ButtonTheme`. I filed https://github.com/bdlukaa/fluent_ui/issues/1139 to add the `.style` property to the drop down button widget.
